### ISSUE TITLE
Revisit usage of XMPP_SERVER with multiple values

### ISSUE THREAD

**Issue by saghul** [jitsi/docker-jitsi-meet#1286]
In some places we currently configure just one:

https://github.com/jitsi/docker-jitsi-meet/blob/95af77896d8c9e27c91c69846aa5aab1f0d74452/jicofo/rootfs/defaults/jicofo.conf#L137
https://github.com/jitsi/docker-jitsi-meet/blob/88bb1bccd09231766f94e7ef2dd3eb977e446a72/jigasi/rootfs/defaults/sip-communicator.properties#L120
https://github.com/jitsi/docker-jitsi-meet/blob/f40a8d56adce68666f4a1e18638f0fde37d39ea8/jicofo/rootfs/etc/services.d/jicofo/run#L6

etc.

**Comment by saghul** [comment]
Ping @aaronkvanmeerten

**Comment by aaronkvanmeerten** [comment]
Yes, it's clear that some of our automation will not work as expected with a list in XMPP_SERVER.  Jigasi in particular I'll need @damencho to help me with, as I'm not quite clear which settings needs to be set to what here:

org.jitsi.jigasi.xmpp.acc.SERVER_ADDRESS
vs
net.java.sip.communicator.impl.protocol.jabber.acc{{ $index }}.SERVER_ADDRESS

For the rest, I believe we may fall back to assuming the first one in the list anytime a singleton would be expected.  In general the docker-compose (or any shared environment config where XMPP_SERVER is set to the same value for all types of containers launched) will not work with multiple XMPP_SERVER set out of the box.
However, other container orchestration systems can use each container type that DOES allow it (JVB, Jigasi (once we fix it), Jibri), and provide a list to get the multiple connections behavior out of these components.  That's my main desired goal from supporting a list in XMPP_SERVER, not getting docker-compose to be able to have N prosody and N jicofo containers out of the box.
